### PR TITLE
Adding eth JSON RPC calls documentation available

### PR DESCRIPTION
**Pull request by eduardonunesp**

### CODE REVIEW

**Comment by eduardonunesp** [comment]
I think people that have a basic knowledge about Ethereum RPC calls, yup it's more like an "internal" thing, but I guess it's also is't a bad thing to have more doc about it

File: website/sidebars.json [hunk 1]
```diff
@@ -1,15 +1,15 @@
 {
   "docs": {
     "Prerequisites": ["prereqs", "prereqs-ubuntu", "prereqs-windows"],
-    "Javascript Client SDK": ["loom-js-quickstart", "web3js-loom-provider-truffle"],
+    "Javascript Client SDK": ["loom-js-quickstart", "web3js-loom-provider-truffle", "json-rpc-methods"],
```
Comment (mattkanwisher) [inline website/sidebars.json]: my thought is we should have an advanced or internal section to the docs

Comment (eduardonunesp) [reply website/sidebars.json]: Yeah probably is a good idea to put into a more advanced area, any suggestion?

**Comment by mattkanwisher** [comment]
Just make a new section

**Comment by eduardonunesp** [comment]
@mattkanwisher New section added "EVM JSON RPC"

**Pull request merged**